File: SeminarskiGymFit/GymFit/GymFit.Infrastructure.Interfaces/SearchObjects/UserPackageSearchObject.cs
﻿using System.Security.Principal;

namespace GymFit.Infrastructure.Interfaces
{
    public class UserPackageSearchObject : BaseSearchObject
    {
        public bool? expired { get; set; }

        public int packageId { get; set; }

        public int userId { get; set; }

        public DateTime? fromDate { get; set; }

        public DateTime? toDate { get; set; }

    }
}

File: SeminarskiGymFit/UI/gymfit_admin/lib/models/searchObjects/user_package_search_object.dart
class UserPackageSearchObject {
  bool? expired;
  int? PageNumber;
  int? PageSize;
  int? userId;
  int? packageId;
  DateTime? fromDate;
  DateTime? toDate;

  UserPackageSearchObject(
      {this.PageNumber,
      this.PageSize,
      this.expired,
      this.userId,
      this.packageId,
      this.fromDate,
      this.toDate});

  UserPackageSearchObject.fromJson(Map<String, dynamic> json) {
    expired = json['expired'];
    userId = json['userId'];
    packageId = json['packageId'];
    PageNumber = json['PageNumber'];
    PageSize = json['PageSize'];
  }

  Map<String, dynamic> toJson() {
    final data = <String, dynamic>{};
    data['expired'] = expired;
    data['PageNumber'] = PageNumber;
    data['PageSize'] = PageSize;
    return data;
  }
}
